redhat-appstudio/remote-secret
# Check remote cluster client before trying to deploy targets

### What does this PR do?

- {SUBJ}
- Add info message about connection failure
- Reduced amount of information that add to the status of RemoteSecret CR

### Screenshot/screencast of this PR



### What issues does this PR fix or reference?

https://issues.redhat.com/browse/SVPI-651

### How to test this PR?

Apply the file:

```
---
kind: Secret
apiVersion: v1
metadata:
 name: test-remote-kubeconfig
type: managed-gitops.redhat.com/managed-environment
data:
 kubeconfig: >-
   YXBpVmVyc2lvbjogdjEKY2x1c3RlcnM6Ci0gY2x1c3RlcjoKICAgIGluc2VjdXJlLXNraXAtdGxzLXZlcmlmeTogdHJ1ZQogICAgc2VydmVyOiBodHRwczovL2FwaS5jbHVzdGVyLTJzN2tnLjJzN2tnLnNhbmRib3gyMzEzLm9wZW50bGMuY29tOjY0NDMKICBuYW1lOiBhcGktY2x1c3Rlci0yczdrZy0yczdrZy1zYW5kYm94MjMxMy1vcGVudGxjLWNvbTo2NDQzCmNvbnRleHRzOgotIGNvbnRleHQ6CiAgICBjbHVzdGVyOiBhcGktY2x1c3Rlci0yczdrZy0yczdrZy1zYW5kYm94MjMxMy1vcGVudGxjLWNvbTo2NDQzCiAgICBuYW1lc3BhY2U6IGRlZmF1bHQKICAgIHVzZXI6IGFkbWluL2FwaS1jbHVzdGVyLTJzN2tnLTJzN2tnLXNhbmRib3gyMzEzLW9wZW50bGMtY29tOjY0NDMKICBuYW1lOiBkZWZhdWx0L2FwaS1jbHVzdGVyLTJzN2tnLTJzN2tnLXNhbmRib3gyMzEzLW9wZW50bGMtY29tOjY0NDMvYWRtaW4KY3VycmVudC1jb250ZXh0OiBkZWZhdWx0L2FwaS1jbHVzdGVyLTJzN2tnLTJzN2tnLXNhbmRib3gyMzEzLW9wZW50bGMtY29tOjY0NDMvYWRtaW4Ka2luZDogQ29uZmlnCnByZWZlcmVuY2VzOiB7fQp1c2VyczoKLSBuYW1lOiBhZG1pbi9hcGktY2x1c3Rlci0yczdrZy0yczdrZy1zYW5kYm94MjMxMy1vcGVudGxjLWNvbTo2NDQzCiAgdXNlcjoKICAgIHRva2VuOiBzaGEyNTZ+MGxwLWdSMXI0T3g2RGJpVm5xX3p2bnp4T2VreUo3MnEwUVpYNTBmeGw1dw==
---
apiVersion: appstudio.redhat.com/v1beta1
kind: RemoteSecret
metadata:
 name: ${OBJ_NAME}
spec:
 targets:
   - apiUrl: https://api-cluster-2s7kg-2s7kg-sandbox2313-opentlc-com:6443
     clusterCredentialsSecret: test-remote-kubeconfig
     namespace: default
 secret:
   name: demo-secret-multiple-environment
   type: Opaque
stringData:
 username: Z2VuYQ==
 password: Z2VuYQ==
```

make sure no big amount of log errors present in logs
